Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -198,7 +198,7 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
 
     !!! caution
         Neither Rook, nor Ceph, prevent the creation of a cluster where the replicated data (or Erasure Coded chunks) can be written safely. By design, Ceph will delay checking for suitable OSDs until a write request is made and this write can hang if there are not sufficient OSDs to satisfy the request.
-* `deviceClass`: Sets up the CRUSH rule for the pool to distribute data only on the specified device class. If left empty or unspecified, the pool will use the cluster's default CRUSH root, which usually distributes data over all OSDs, regardless of their class.
+* `deviceClass`: Sets up the CRUSH rule for the pool to distribute data only on the specified device class. If left empty or unspecified, the pool will use the cluster's default CRUSH root, which usually distributes data over all OSDs, regardless of their class. If `deviceClass` is specified on any pool, ensure the it is added to every pool in the cluster, otherwise Ceph will warn about pools with overlapping roots.
```

Comment:
Since now Pacific, leaving empty deviceClass makes autoscaler unhappy - i guess we need to highlight that parameter should be set, otherwise autoscaler will not work. In the same time, we can't mark parameter as required since it is not backward compatible. So i think, something like:

* `deviceClass`: Sets up the CRUSH rule for the pool to distribute data only on the specified device class. Since Pacific, if left empty or unspecified, the pool will use the cluster's default CRUSH root, but pool pg autoscaler will stop working for all such pools and affected device classes, which have pg from that pool.

Reply:
I'm understanding the deviceClass is only required for all pools if any pools sets the deviceClass. If no pool sets the deviceClass, then this error won't be raised, right? It doesn't happen by default, or else all rook users would be complaining about this issue.